src/commands/commit/withClient/index.js

```diff
@@ -5,9 +5,18 @@ import chalk from 'chalk'
 import isHookCreated from '@utils/isHookCreated'
 import configurationVault from '@utils/configurationVault'
 import { type Answers } from '../prompts'
+import isGitRepository from '@utils/isGitRepository'
 
 const withClient = async (answers: Answers): Promise<void> => {
   try {
+    if(!isGitRepository()) {
```

Comment:
I would probably avoid handling this through a helper and instead improve the way we show errors to users.

This way we can catch all exceptions handling them gracefully